src/Listener/ApiPaginationListener.php

```diff
@@ -58,13 +58,19 @@ public function beforeRender(Event $event)
             return;
         }
 
+        $paginationLimit = $pagination['limit'];
+        if ($pagination['limit'] == null) {
+            $paginationLimit = $controller->paginate['limit'];
+        }
+
         $paginationResponse = [
             'page_count' => $pagination['pageCount'],
             'current_page' => $pagination['page'],
             'has_next_page' => $pagination['nextPage'],
             'has_prev_page' => $pagination['prevPage'],
             'count' => $pagination['count'],
-            'limit' => $pagination['limit']
+            'limit' => $paginationLimit,
+            'per_page' => $pagination['perPage']
```

Comment:
What's this for? The non-existing key is causing tests to fail https://travis-ci.org/FriendsOfCake/crud/jobs/120870444#L327